### PR TITLE
Added new Country class with test

### PR DESCRIPTION
**Pull request by tanmay380**

### CODE REVIEW

**Comment by tanmay380** [comment]
@gabrielbarker When will u be merging these pull requests?

File: test/categories/test_country.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+  def test__get_values__returns_list(self):
+    self.assertIsInstance(self.country.get_values(), list)
+    self.assertGreater(len(self.country.get_values()), 0)
+
+  def test__get_values__each_string_is_correct(self):
+    for value in self.country.get_values():
+      self.assertFalse(' ' in value)
+
+  def test__get_values__is_valid(self):
```
Comment (gabrielbarker) [inline test/categories/test_country.py:18]: This test performs all the validation of the two tests above, so you can remove `test__get_values__returns_list` and `test__get_values__each_string_is_correct` completely

**Review by gabrielbarker — CHANGES_REQUESTED** [review]
Perfect apart from that one small comment 🙂
I will merge once that is done

**Review by gabrielbarker — APPROVED** [review]

**Review by gabrielbarker — APPROVED** [review]

**Comment by gabrielbarker** [comment]
Great work @tanmay380 , thanks a lot!

**Comment by tanmay380** [comment]
I'll add more categories. Dont worry.



[image: Mailtrack]
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
Sender
notified by
Mailtrack
<https://mailtrack.io?utm_source=gmail&utm_medium=signature&utm_campaign=signaturevirality5&>
01/10/20,
14:15:31

**Pull request merged**